spec/barbeque/executor/docker_spec.rb

```diff
@@ -375,6 +413,59 @@
             expect(job_execution).to be_retried
           end
         end
+
+        context 'when retried job fails for retry limit of times' do
+          let(:slack_client) { double('Barbeque::SlackClient') }
+          let(:container_id2) { '40fa4fcb316f90f65d70717f66357e065747f8c216f392817bb8237f51dc416e' }
+          let(:job_retry2) { FactoryBot.create(:job_retry, job_execution: job_execution, status: :pending) }
+
+          before do
+            container_info['State']['ExitCode'] = 1
```

Comment:
`container_info['State']['ExitCode']` is already set to 1 at https://github.com/cookpad/barbeque/pull/77/files/f06433277c02d70f68d36f984837cd5a43dbe1fc..27623729598bab3f8a0afac66f0df466ce61a9d6#diff-cdd9428b8bcd0f6d9b5128e2e266911fR366 ?

Reply:
Removed 🙈  https://github.com/cookpad/barbeque/pull/77/commits/df62e5147aad21cbeb46d225aa4e5a08c2233341